subcommands/waves/init.go

```diff
@@ -145,6 +150,31 @@ func doInitWave(cmd *cobra.Command, args []string) {
 	} else {
 		subcommands.DieNotNil(api.FactoryCreateWave(factory, &wave), "Failed to create a wave")
 	}
+
+	if !hasStaticDelta(new_targets) {
+
+		fmt.Print(`
+WARNING: You created a wave for a target version without static deltas.
+
+We recommend that you generate static deltas for your production targets to optimize
```

Comment:
I've tested this change, it works as expected.

I am not sure that the wording of this message as well as the message at #32 conveys accurately what needs to be done, though. I'll let the support team representative @vanmaegima to judge on it.

In any case, I think we can improve it at the next iteration/later, so I think it's good to go.